policyengine_us/parameters/gov/states/wi/tax/income/credits/childcare_expense/fraction.yaml

```
@@ -10,6 +10,9 @@ metadata:
       href: https://www.revenue.wi.gov/TaxForms2022/2022-Form1-Inst.pdf#page=17
     - title: 2023 Form 1 instructions
       href: https://www.revenue.wi.gov/TaxForms2023/2023-Form1-Inst.pdf#page=18
+    - title: 2024 Wisconsin Statutes Taxation. 71.07 (9g) (b) 2.
```

As (2) includes the 2022-23 values we have.
```suggestion
    - title: 2024 Wisconsin Statutes Taxation. 71.07 (9g) (b)
```